coding-concepts.js

```diff
@@ -8,36 +8,38 @@
 // --------------------1) What will this log?
 
 const cohort = "Charlie 2023"
-// console.log(cohort.split(""))
+// console.log(cohort.split(" "))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: all letters in the string "Charlie 2023" in a separate string.
+// b) Verify and explain: .split is a built in method that will separate a string by each letter when it has the function of "". " "(quotations with a space in the middle) will return each word split into its own string.
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
   `Hello, ${name}!`
 }
 // console.log(greeter("LEARN Student"))
+//output: undefined
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: undefined
+// b) Verify and explain: The variable greeter does not have a value defined, meaning that name is equal to nothing and so the greeter variable being called returns undefined.
```

Comment:
`greeter` is a function, and `name` is the parameter that is being defined. The reason it returns undefined is because there is no `return` in the function!